.github/gh_matrix_builder.py

```diff
@@ -141,10 +141,22 @@ def macos_config(overrides):
 # partialize_finalize is ignored due to #4937
 pg15_installcheck_args = "IGNORES='partialize_finalize'"
 
+# transparent_decompress_chunk-13 is ignored due to #5118
+pg13_installcheck_args = (
+    "IGNORES='bgw_db_scheduler bgw_db_scheduler_fixed transparent_decompress_chunk-13'"
```

Comment:
bgw tests are generally flaky, but we shouldn't be ignoring them in this PR I think.